src/main/java/uk/gov/gchq/palisade/exception/Status.java

```diff
@@ -101,10 +103,12 @@ public static Status fromStatusCode(final int statusCode) {
                 .orElseThrow(RuntimeException::new);
     }
 
+    @Generated
     public int getStatusCode() {
         return statusCode;
     }
 
+    @Generated
     public String getReason() {
         return reason;
     }
```

Comment:
Add toString, hashCode and equals methods

Reply:
See previous comment about Status class